Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -320,9 +323,15 @@ public class PythonCompiler : SinglePassCompiler {
       var cw = (ClassWriter)CreateClass(IdProtect(nt.EnclosingModuleDefinition.CompileName), IdName(nt), nt, wr);
       var w = cw.MethodWriter;
       var udt = UserDefinedType.FromTopLevelDecl(nt.tok, nt);
-      var d = TypeInitializationValue(udt, wr, nt.tok, false, false);
-
-      w.WriteLine($"default = staticmethod(lambda: {d})");
+      if (nt.WitnessKind == SubsetTypeDecl.WKind.Compiled) {
+        w.WriteLine("@staticmethod");
+        var block = w.NewBlockPy("def default():");
+        var wStmts = block.Fork();
+        block.Write("return ");
+        TrExpr(nt.Witness, block, false, wStmts);
+      } else {
+        w.WriteLine($"default = staticmethod(lambda: {TypeInitializationValue(udt, wr, nt.tok, false, false)})");
```

Comment:
`staticmethod(lambda: … )` isn't the more readable — why not use the same form for both?
(Or use a property in the first case and a constant on the second case)